VideoLocker/res/layout/panel_message_with_check.xml

```diff
@@ -13,6 +13,7 @@
 
     <ImageView
         android:id="@+id/message_indicator"
+        android:contentDescription="@null"
```

Comment:
This image is a checkmark and is displayed when we properly signed up with Facebook or Google+. The checkmark itself is never focusable in accessibility so content description can be null.